p4broker/plan.sh

```diff
@@ -0,0 +1,39 @@
+pkg_name=p4broker
+pkg_origin=core
+pkg_version="18.1"
+pkg_maintainer="Daniel Hagen <daniel.b.hagen@gmail.com>"
+pkg_license=("Apache-2.0")
+pkg_source="http://www.perforce.com/downloads/perforce/r${pkg_version}/bin.linux26x86_64/p4broker"
+pkg_filename="p4broker"
+pkg_shasum="5be8b8e6c327e590d83c0597d27120fee1ff628f568342309bcde4b5b6e669d9"
+pkg_upstream_url="https://www.perforce.com/downloads/helix-broker-p4broker"
+pkg_deps=(core/glibc)
+pkg_build_deps=(core/make core/gcc core/patchelf)
+pkg_bin_dirs=(bin)
+pkg_exports=(
+  [host]=srv.address
+  [port]=srv.port
+)
+pkg_exposes=(port)
+pkg_description="Helix' P4Broker daemon service for proxying requests through to a Helix PerForce server."
+
+do_build() {
+	# No build needed.
+	return 0
+}
+
+do_unpack() {
+	# No unpack needed
+	return 0
+}
+
+do_install() {
```

Comment:
It's generally considered a safe practice when using patchelf to override `do_strip` to not strip. Ex: 
```
do_strip() {
  return 0
}
```